apache-arrow/README.md

```diff
@@ -1,3 +1,11 @@
+### Why We Mirror Apache Arrow
+
+Groonga depends on Apache Arrow. However, the official Apache Arrow repository has been unstable at times, causing issues for Groonga users. To resolve this situation, we have been providing a mirrored version for Groonga users.
+
+Going forward, we plan to discontinue the mirror except in the following cases.
```

Comment:
We would like to stop mirroring.
Therefore, I think it is sufficient to just state that we intend to stop mirroring once the Apache Arrow repository is stable.

Reply:
Ah. I see.
My opinion is the same abetomo.

Reply:
Thank you for reviewing.
fix: [0a767c8](https://github.com/groonga/packages.groonga.org/pull/40/commits/0a767c8c66f638e36900601061fda3545c301852) The key point is to mention our intention to stop mirroring. Therefore, I have removed unnecessary context.